timesketch/lib/tasks.py

```diff
@@ -484,3 +486,52 @@ def run_csv_jsonl(source_file_path, timeline_name, index_name, source_type):
     _set_timeline_status(index_name, status='ready')
 
     return index_name
+
+
+@celery.task(track_started=True, base=SqlAlchemyTask)
+def run_mans(source_file_path, timeline_name, index_name, source_type):
+    """Create a Celery task for processing mans file.
+
+    Args:
+        source_file_path: Path to mans file.
+        timeline_name: Name of the Timesketch timeline.
+        index_name: Name of the datastore index.
+        source_type: Type of file, csv or jsonl.
+
+    Returns:
+        Name (str) of the index.
+    """
+    # Log information to Celery
+    message = 'Index timeline [{0:s}] to index [{1:s}] (source: {2:s})'
+    logging.info(message.format(timeline_name, index_name, source_type))
+
+    try:
+        mans_to_es_path = current_app.config['MANS_TO_ES_PATH']
```

Comment:
I'm not sure why this is being done...

mans_to_es seems to be a python project, why on earth are you requiring someone to install the toola dn then use supbrocess to run it, rather than just import it as a module (have it in the requirements and then import iit)

looks to me all that the command line tool does is to parse the command line arguments and then run 

```
            mte = MansToEs(args)
            mte.extract_mans()
            mte.parse_manifest()
            mte.parse_hits()
            mte.process()
            mte.to_elastic()
            mte.delete_temp_folder()
```

so you might as well just import the library and do the same here, and avoid this whole subprocess

Reply:
Tried to follow the same behavior of plaso, when called psort.py
If you think is better to use library, we can change the approach.
We tought in this way it was easier to add features or fixes at our side without opening pr on timesketch repo.